inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -166,7 +166,23 @@ void GNAModelSerial::Import(void *basePointer,
         InferenceEngine::OutputsDataMap& outputsDataMap) {
     is.exceptions(std::istream::failbit);
 
+    for (auto inputIndex = 0; inputIndex < modelHeader.nInputs; inputIndex++) {
+        uint32_t nameSize = 0;
+        readNBits<32>(nameSize, is);
+        std::vector<char> inName(nameSize);
```

Comment:
You can use std::string instead of std::vector<char> and resize it

Reply:
I can't use std::string because std:: string can't be converted to char*.

Reply:
you can use &inName[0]